### ISSUE TITLE
Error on https://www.games68.com/games.php?id=2108874

### ISSUE THREAD

**Issue by whobiggs** [ruffle-rs/ruffle#7600]
# Error Info
Error name: ReferenceError
Error message: WebAssembly is not defined
Error stack:
```
ReferenceError: WebAssembly is not defined
    at https://www.games68.com/include/ruffle/ruffle.js:1:32014
    at e (https://www.games68.com/include/ruffle/ruffle.js:1:32588)
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: games/50b3c5edb5190.swf
Param movie: games/50b3c5edb5190.swf
Param quality: high
Param allowScriptAccess: always
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined

# Page Info
Page URL: https://www.games68.com/games.php?id=2108874

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.81 Safari/537.36 Edg/104.0.1293.47
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-17
Channel: nightly
Built: 2021-01-17T01:05:59.152Z
Commit: c2b768c453f281a988c71a829e7a71bb106302fa

**Comment by n0samu** [comment]
This problem is caused by Microsoft Edge's "Enhance your security on the web" feature. [See the FAQ for a solution to the problem](https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users#im-trying-to-use-ruffle-on-microsoft-edge-but-i-always-get-a-something-went-wrong-error).

**Comment by whobiggs** [comment]
Thanks but apparently there is a Trojan in it.
RegardsDavid Beacham

 

    On Monday, 8 August 2022 at 11:55:22 BST, nosamu ***@***.***> wrote:  
 
 


Closed #7600 as not planned.
